src/wp-includes/rest-api/endpoints/class-wp-rest-themes-controller.php

```diff
@@ -409,6 +547,11 @@ public function get_item_schema() {
 					'type'        => 'string',
 					'readonly'    => true,
 				),
+				'status'         => array(
```

Comment:
@TimothyBJacobs Should this be a boolean?

Reply:
I think we're already committed to `status=active`.

Reply:
Yeah, shame :(